iree/compiler/Codegen/SPIRV/SPIRVVectorize.cpp

```diff
@@ -33,21 +33,29 @@ namespace mlir {
 namespace iree_compiler {
 namespace {
 
-Optional<SmallVector<int64_t, 4>> getSPIRVNativeVectorSize(Operation *op) {
+int getNativeVectorSize(int64_t size) {
```

Comment:
Should we fall back to 1 if it doesn't divide by 2? Otherwise we will have the same problem that the transfer op won't get broken up.